docs/usage.md

```diff
@@ -23,12 +23,19 @@ A relatively trivial example of an analysis can be found in this repo [here](../
 
 Enclave containers can access the following data CSV files:
 
-* [assessments.csv](#assessmentscsv)
-* [courses.csv](#coursescsv)
-* [enrollments.csv](#enrollmentscsv)
-* [grades.csv](#gradescsv)
-* [oneroster_demographics.csv](#oneroster_demographicscsv)
-* [users.csv](#userscsv)
+- [RAISE Enclave Usage](#raise-enclave-usage)
+  - [Operational model](#operational-model)
+  - [Data model](#data-model)
+  - [`assessments.csv`](#assessmentscsv)
+  - [`courses.csv`](#coursescsv)
+  - [`enrollments.csv`](#enrollmentscsv)
+  - [`grades.csv`](#gradescsv)
+  - [`oneroster_demographics.csv`](#oneroster_demographicscsv)
+  - [`users.csv`](#userscsv)
+  - [`quiz_questions.csv`](#quiz_questionscsv)
+  - [`quiz_question_contents.csv`](#quiz_question_contentscsv)
+  - [`quiz_multichoice_answer.csv`](#quiz_multichoice_answercsv)
```

Comment:
Can we re-order these in this list and the content to come before `users.csv` so things are roughly alphabetical?